umbraco/Umbraco.UI
# Bugfix: Toasts should restart autoclose after mouse leaves

## Description

When using a notification container but use the auto-close option on each notification element. The notifications don't auto-close after one of the notifications has hovered.

This request also changes the event types from **mouseover/mouseout** to **mouseenter/mouseleave** to ensure that the logic behind the events are only fired once, i.e. if you move your mouse around _while hovering_ the logic would keep being executed each tick.

Fixes [AB#17243](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/17243)

## Types of changes




## How to test?

1. Find the story for the notification container
2. remove the auto close attribute from the container element
3. set an autoClose property on a notification in the add function
4. When you open a notification they will auto close as expected
5. If you hover a notification and leave. None of the notifications auto-close. If you manually close the notification and open new ones, they won't auto-close either.
6. Refresh the browser and the notification will auto close again until you hover one of them.

## Checklist
